### DISCUSSION TITLE
Tab Size

### DISCUSSION THREAD

**Discussion by chrsBlank** [zauberzeug/nicegui#741] [Q&A] [answered]
### Question

Hi, i was wondering how to set the size of tabs.
```py
with ui.tab_panels(tabs, value='Home'):
    with ui.tab_panel('Home'): ...
    with ui.tab_panel('Statistics'): ...
```
and the size is like that
![image](https://user-images.githubusercontent.com/36511746/231113617-071e4e30-97a4-4387-8be9-3387f460e60d.png)

and on mobile it uses the whole screen

and this is without tabs ( on mobile still uses the whole screen )
![image](https://user-images.githubusercontent.com/36511746/231113986-b574814d-2140-4b7b-9522-8ebf81c68c4a.png)

how can i set the tab size?

**Comment by phoskee** [comment]
I think you could solve it by setting the width of `ui.tab_panels`

```py
with ui.tab_panels(tabs, value='Home').style('width: 100%'):
```

This element uses the class `q-tab-panels q-panel-parent` which seems not to include `width: 100%`

**Reply by chrsBlank** [reply]
nice, that solved my issue, thank you, thought i still have the buttons side by side instead of stacked like without the tabs.
sorry for the 'pretty easy' questions, i dont have much expirience with front end design etc

**Reply by phoskee** [reply, marked as answer]
Without your code I cannot help you, within the limits of my skills
But, I think you can achieve the goal with this:

```py
with ui.column():
    ui.button(text='Button1')
    ui.button(text='Button2')
    ui.button(text='Button3')
```
